make/proto.mk

```diff
@@ -27,41 +27,41 @@ DOCKER_BUF := docker run -v $(shell pwd):/workspace --workdir /workspace bufbuil
 # Uncomment next lines while working on pulsar and swagger functionality
 
 
-# proto-lint:
-# 	@$(DOCKER_BUF) lint --error-format=json
+proto-lint:
+	@$(DOCKER_BUF) lint --error-format=json
 
-# proto-check-breaking:
-# 	@$(DOCKER_BUF) breaking --against https://github.com/cheqd/cheqd-node.git#branch=main
+proto-check-breaking:
+	@$(DOCKER_BUF) breaking --against https://github.com/cheqd/cheqd-node.git#branch=main
 
-# GOGO_PROTO_URL           = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
-# GOOGLE_PROTO_URL         = https://raw.githubusercontent.com/googleapis/googleapis/master
-# COSMOS_PROTO_URL         = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/proto/cosmos
-# COSMOS_ORM_PROTO_URL     = https://raw.githubusercontent.com/cosmos/cosmos-sdk/orm/v1.0.0-alpha.10/proto/cosmos
+GOGO_PROTO_URL           = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
+GOOGLE_PROTO_URL         = https://raw.githubusercontent.com/googleapis/googleapis/master
+COSMOS_PROTO_URL         = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/proto/cosmos
```

Comment:
If required (and I don't think it is), make this match Cosmos SDK version in go.mod

Reply:
Yep, not required. I was able to generate the docs with these.